docs/api.html

```diff
@@ -169,7 +169,7 @@
 <h3><a id="streamsapi" href="#streamsapi">2.3 Streams API</a></h3>
 
 As of the 0.10.0 release we have added a new client library named <b>Kafka Streams</b> to let users implement their stream processing
-applications with data stored in Kafka topics. Kafka Streams is considered alpha quality and its public APIs are likely to change in
+applications with data stored in Kafka topics. Kafka Streams is a new library and its public APIs are likely to change in
```

Comment:
The updated section also has the following paragraph at the very end:

> Examples showing how to use this library are given in the javadocs (note those classes annotated with @InterfaceStability.Unstable, indicating their public APIs may change without backward-compatibility in future releases). 

So perhaps we should reword/merge the section to sth like the following (including some tweaks to explaining what Kafka Streams is):

> As of the 0.10.0 release we have added a stream processing engine to Apache Kafka called **Kafka Streams**, which is a client library that lets users implement their own stream processing applications for data stored in Kafka topics. You can use Kafka Streams from within your Java applications by adding a dependency on the kafka-streams jar using the following example maven co-ordinates (you can change the version numbers with new releases):
>
> ```
	<dependency>
	    <groupId>org.apache.kafka</groupId>
	    <artifactId>kafka-streams</artifactId>
	    <version>0.10.0.0</version>
	</dependency>
> ```
>
> Examples showing how to use this library are given in the [javadocs](http://kafka.apache.org/0100/javadoc/index.html?org/apache/kafka/streams/KafkaStreams.html).
>
> Please note that Kafka Streams is a new component of Kafka, and its public APIs may change in future releases.  We use the `@InterfaceStability.Unstable` annotation to denote classes whose APIs may change without backward-compatibility in future releases.